wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceStartAttemptsMeterBinder.java

```diff
@@ -40,13 +43,22 @@ public void bindTo(MeterRegistry registry) {
         Counter.builder(workspaceMetric("starting_attempts.total"))
             .description("The count of workspaces start attempts")
             .register(registry);
+    startingDebugCounter =
+        Counter.builder(workspaceMetric("starting_attempts.total"))
+            .tags(withStandardTags("debug"))
+            .description("The count of workspaces start attempts in debug mode")
+            .register(registry);
 
     // only subscribe to the event once we have the counters ready
     eventService.subscribe(
         event -> {
           if (event.getPrevStatus() == WorkspaceStatus.STOPPED
               && event.getStatus() == WorkspaceStatus.STARTING) {
             startingCounter.increment();
```

Comment:
You increment just one of the counters. The total is produced by prometheus automagically. So basically, we could have: `starting_attempts.total#debug=false` and `starting_attempts.total#debug=true` and just increment one of them here. We would not have an untagged counter at all.

Reply:
hopefuly tags are ok now